engine/inc/loaders/texture/png_loader.hpp

```diff
@@ -3,7 +3,7 @@
 #   |     \/   ____| |___|
 #   |     |   |   \  |   |
 #-----------------------------------------------------------------------
-# Copyright 2020, tyra - https://github.com/h4570/tyra
+# Copyright 2022, tyra - https://github.com/h4570/tyra

```

Comment:
Hmm I dont get it.
We should be consistent with this
`2022` or `2020-2022` (the same for other places)